Build.proj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TraversalGlobalProperties Condition="'$(BuildAllConfigurations)' != 'true'">BuildTargetFramework=$([MSBuild]::ValueOrDefault('$(BuildTargetFramework)', '$(NetCoreAppCurrent)'))</TraversalGlobalProperties>
+    <ScriptExt Condition="'$(OS)' == 'Windows_NT'">.cmd</ScriptExt>
+    <ScriptExt Condition="'$(OS)' != 'Windows_NT'">.sh</ScriptExt>
```

Comment:
This doesn't belong here as other project won't be able to access the property. Instead you want to add it either to https://github.com/dotnet/runtime/blob/master/eng/Configurations.props or https://github.com/dotnet/runtime/blob/master/Directory.Build.props.

Also the `ScriptExt` name doesn't make sense as there aren't just cmd and .sh scripts. As an example, on Windows we more often use ps1 scripts instead of cmd ones.